package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "login.dfe.manage",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
Services are currently using release branches rather than tags so you probably didn't see that the latest version number would be (https://github.com/DFE-Digital/login.dfe.manage/tree/release/1.2.2).

I appreciate that this is a little inconsistent and the version numbers of the services have not been consistently bumped.

If we update this I think it becomes 1.3.0.